containers/ome-tiff-offsets/context/main.py

```diff
@@ -3,34 +3,39 @@
 from pathlib import Path
 from os import makedirs
 import json
+from os.path import dirname, exists, splitext
 
 from skimage.external import tifffile
 
-
 def get_offsets(tiff_filepath):
     with tifffile.TiffFile(tiff_filepath) as tif:
         offsets = [page._offset for page in tif.pages]
     return offsets
 
 def main(input_dir, output_dir):
     makedirs(output_dir, exist_ok=True)
-    for input_path in glob(input_dir + '/*.ome.tiff') + glob(input_dir + '/*.ome.tif'):
-        offsets = get_offsets(input_path)
-        # Truncate `.ome.tiff`.
-        input_name = Path(input_path).with_suffix('').with_suffix('').name
-        output_path = Path(output_dir) / str(input_name + '.offsets.json')
+
+    # Find all OME.TIFFs in the input directory.
+    for input_path in input_dir.glob('**/*.ome.tif'):
```

Comment:
@ilan-gold : Is it ok not to handle `.tiff`?

Reply:
See comment - I would like it to be more general.

Reply:
@mccalluc and @ilan-gold added back support for ome.tiff. Calling

```
cwltool --debug ome-tiff-offsets.cwl --input_dir /Users/icaoberg/Pictures/89f573d6b32066942b33bc476df52a7a/ometiff-pyramids
```

on

```
tree ~/Pictures/89f573d6b32066942b33bc476df52a7a/ometiff-pyramids
/Users/icaoberg/Pictures/89f573d6b32066942b33bc476df52a7a/ometiff-pyramids
├── 0d923937-551c-4ff5-958c-5fe9d008eae3
│   ├── multi-channel-2.ome.tiff
│   └── multi-channel.ome.tif
├── 2543ce47-162a-477b-91f7-0a1ca6b0e5ed
│   ├── multi-channel-2.ome.tiff
│   └── multi-channel.ome.tif
├── 27f36f8b-7054-4c96-98d4-c0b9929c6403
│   ├── multi-channel-2.ome.tiff
│   └── multi-channel.ome.tif
├── 75e838cd-04ee-4723-a2b8-9a7444d37f5d
│   ├── multi-channel-2.ome.tiff
│   └── multi-channel.ome.tif
└── c49e24f1-2fd1-4e34-9f98-4ac68d48d39d
    ├── multi-channel-2.ome.tiff
    └── multi-channel.ome.tif

5 directories, 10 files
```

produces

```
tree ./output_offsets
./output_offsets
├── 0d923937-551c-4ff5-958c-5fe9d008eae3
│   ├── multi-channel-2.offsets.json
│   └── multi-channel.offsets.json
├── 2543ce47-162a-477b-91f7-0a1ca6b0e5ed
│   ├── multi-channel-2.offsets.json
│   └── multi-channel.offsets.json
├── 27f36f8b-7054-4c96-98d4-c0b9929c6403
│   ├── multi-channel-2.offsets.json
│   └── multi-channel.offsets.json
├── 75e838cd-04ee-4723-a2b8-9a7444d37f5d
│   ├── multi-channel-2.offsets.json
│   └── multi-channel.offsets.json
└── c49e24f1-2fd1-4e34-9f98-4ac68d48d39d
    ├── multi-channel-2.offsets.json
    └── multi-channel.offsets.json

5 directories, 10 files
```